app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,71 @@
+class TasksController < ApplicationController
+  def index
+    @all_tasks = Task.all
+    render :index
+  end
+
+  def show
+    @task = Task.find(params[:id])
+    if @task.completed_date.present?
+      @task_css = "completed"
+    else
+      @task_css = "not_completed"
+    end
+    render :show
+  end
+
+  def edit
+    @task = Task.find(params[:id])
+    render :edit_task
+  end
+
+  def update
+    @task = Task.find(params[:id])
+    @task.update(form_params[:task])
+
+    redirect_to action: :index
+  end
+
+  def mark_done
+    @task = Task.find(params[:id])
+    @task.update(completed_date: Time.new)
+    redirect_to action: :index
+  end
+
+  def new
+    @task = Task.new
+    @people = Person.all
+  end
+
+  def create
+    @task = Task.new(form_params[:task])
+    @task.save
+    redirect_to action: :index
+  end
+
+  def confirm_remove
+    @task = Task.find(params[:id])
+    if @task.completed_date.present?
+      @task_css = "completed"
+    else
+      @task_css = "not_completed"
```

Comment:
We hadn't discussed view helpers. So you mean like this? http://www.rails-dev.com/custom-view-helpers-in-rails-4